src/test/java/teammates/test/cases/datatransfer/FeedbackMcqQuestionDetailsTest.java

```diff
@@ -411,4 +415,91 @@ public void testExtractQuestionDetails_weightsEnabledForGenerateOptions_weightSh
         assertTrue(mcqDetails.getMcqWeights().isEmpty());
         assertEquals(0.0, mcqDetails.getMcqOtherWeight());
     }
+
+    @Test
+    public void testCreateQuestionDetails_mcqTypeQuestion_createdSuccessfully() {
+        HashMap<String, String[]> requestParameters = new HashMap<>();
+
+        requestParameters.put(Const.ParamsNames.FEEDBACK_QUESTION_TEXT, new String[] { "MCQ question" });
+        requestParameters.put(Const.ParamsNames.FEEDBACK_QUESTION_TYPE, new String[] { "MCQ" });
+        requestParameters.put(Const.ParamsNames.FEEDBACK_QUESTION_MCQ_GENERATED_OPTIONS,
+                new String[] {FeedbackParticipantType.NONE.toString()});
+        requestParameters.put(Const.ParamsNames.FEEDBACK_QUESTION_NUMBEROFCHOICECREATED, new String[] { "1" });
+        requestParameters.put(Const.ParamsNames.FEEDBACK_QUESTION_MCQCHOICE + "-0", new String[] { "mcq option" });
+
+        FeedbackQuestionDetails feedbackQuestionDetails =
+                FeedbackQuestionDetails.createQuestionDetails(requestParameters, FeedbackQuestionType.MCQ);
+
+        assertEquals(feedbackQuestionDetails.getQuestionType(), FeedbackQuestionType.MCQ);
+        assertTrue(feedbackQuestionDetails instanceof FeedbackMcqQuestionDetails);
+        assertEquals("MCQ question", feedbackQuestionDetails.getQuestionText());
+    }
+
+    @Test
+    public void testGetQuestionWithExistingResponseSubmissionFormHtml_responsePresent_htmlTagsPresent() {
+        HashMap<String, String[]> requestParameters = new HashMap<>();
+
+        requestParameters.put(Const.ParamsNames.FEEDBACK_QUESTION_TEXT, new String[] { "MCQ" });
```

Comment:
It would be good to create a method to create the parameters that explain what kind of question we are creating? Is it an MCQ question with two options? Three options? What does each parameter mean?

It would also be good to reuse this method across the tests.